src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -476,6 +477,9 @@ private CodeRefactoring FilterOnUIThread(CodeRefactoring refactoring, Workspace
                 }
             }
 
+            private static bool IsTopLevelSuppressionAction(CodeAction action)
+                => action is AbstractTopLevelConfigurationOrSuppressionCodeAction;
```

Comment:
we have AbstractTopLevelConfigurationOrSuppressionCodeAction.  does it need to be abstract?  what are the subclasses of it?  Feels like it could just be the TopLevelConfigurationAction, and the supression/severity items are just nested in that.

Reply:
`AbstractTopLevelConfigurationOrSuppressionCodeAction` represents the top level code action created by each configuration/suppression provider. Then the engine groups them together and wraps all these per-provider top level actions under another topmost level suggested action for that shows up in the light bulb menu.